### PR TITLE
FEATURE: BB-998 Extend pop measure title & alias lookup

### PR DESCRIPTION
**Pull request by martinmosko**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/198/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1436/ : SUCCESS in 4m 11s

File: src/helpers/popHelper.ts [hunk 1]
```diff
@@ -3,17 +3,35 @@ import { VisualizationObject } from '@gooddata/typings';
...
+    const buckets = get<VisualizationObject.IBucket[]>(mdObject, 'buckets', []);
+    const allBucketItems = flatMap<VisualizationObject.BucketItem>(buckets, bucket => bucket.items);
+
+    const measureBucketItems: VisualizationObject.IMeasure[] = [];
+
+    allBucketItems.forEach((bucketItem) => {
+        if (VisualizationObject.isMeasure(bucketItem)) {
+            measureBucketItems.push(bucketItem);
+        }
+    });
```
Comment (strouhaljan) [inline src/helpers/popHelper.ts]: Could you use `reduce` instead of `forEach`?

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/200/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1438/ : SUCCESS in 1m 41s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/598/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/201/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/506/ : SUCCESS in 17m 41s

**Comment by petrjanu** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/31/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1439/ : SUCCESS in 4m 11s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/599/ : SUCCESS in 15m 56s

**Pull request merged**